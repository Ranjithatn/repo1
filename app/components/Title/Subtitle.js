import React from 'react'

const Subtitle = ({
  text,
  type,
  className,
  is,
  as: Hn,
  ...props
}) => {
  return <Hn className={`subtitle is-${is}`} {...props}>{ text }</Hn>;
}

Subtitle.defaultProps = {
  as: 'h1',
  is: '1'
}

export default Subtitle;
