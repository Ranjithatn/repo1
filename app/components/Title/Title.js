import React from 'react'

const Title = ({
  text,
  type,
  className,
  is,
  as: Hn,
  ...props
}) => {
  return <Hn className={`title is-${is}`} {...props}>{ text }</Hn>;
}

Title.defaultProps = {
  as: 'h1',
  is: '1'
}

export default Title;
