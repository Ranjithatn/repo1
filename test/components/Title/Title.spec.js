import React from "react";
import toJson from "enzyme-to-json";
import Title from "../../../app/components/Title/Title";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });

describe('<Title />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <Title  />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });

});
