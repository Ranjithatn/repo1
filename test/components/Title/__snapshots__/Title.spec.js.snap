// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render without crashing 1`] = `
<Title
  as="h1"
  is="1"
>
  <h1
    className="title is-1"
  />
</Title>
`;
